src/runtime_src/core/tools/common/ReportCmcStatus.cpp

```diff
@@ -35,6 +35,7 @@ ReportCmcStatus::getPropertyTree20202( const xrt_core::device * _pDevice,
 {

```

Comment:
Copyright date needs to updated (line 2).  In this case, a new line needs to be added:
`Copyright (c) 2022 Advanced Micro Devices, Inc.`